reference/lib/ReferenceVerifiers.sol

```diff
@@ -121,6 +121,9 @@ contract ReferenceVerifiers is
             return false;
         }
 
+        // Read order status numerator from storage and place on stack.
+        uint256 orderStatusNumerator = orderStatus.numerator;
```

Comment:
This variable is not used atm. Could be used to replace `orderStatus.numerator` below.